### DISCUSSION TITLE
DNS not enabled in all locations

### DISCUSSION THREAD

**Discussion by omdevops99** [learndevopsonline/learndevopsonline#1569] [Q&A] [answered]
Hi All,

DNS name was not enbled in all locations, So again I have to create a new DNS or how ?
![image](https://github.com/learndevopsonline/learndevopsonline/assets/141154551/f78eeb25-0136-48cc-8a33-a47740fdbd4a)

**Comment by sreedharm07** [comment]
@omswetha 
it will take time to enable in all locations.
when you created hosted zone in route 53 ?

**Reply by omdevops99** [reply]
@sreedharm07 
I have been created two or three days back

**Reply by sreedharm07** [reply, marked as answer]
if most of locations are enabled means i think we can use it

**Reply by omdevops99** [reply]
Okay Sure,Thanks!!

**Reply by vikram9491** [reply]
hi  @omswetha  whether your domain name checked in all locations ...? same issue i am getting
